packages/webview_flutter/webview_flutter_android/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 3.1.0
+
+* Adds support for selecting Hybrid Composition on versions 23+. Please use
+  `AndroidWebViewControllerCreationParams.displayWithHybridComposition`.
```

Comment:
Could you add a README entry for this too? I'd like to update webview_flutter to match the structure we're using for google_maps_flutter now, where the main docs for display mode are [in the implementation package readme](https://pub.dev/packages/google_maps_flutter_android#display-mode) and the main package readme just [links to them](https://pub.dev/packages/google_maps_flutter#android) with almost no detail.

That will avoid the problem we have now where the main package readme is documenting things that are actually controlled in an implementation package, so the README gets out of sync with the implementation.